### PR TITLE
Remote debugging support

### PR DESCRIPTION
**Pull request by marcgs**
Adapt docker compose to allow remote debugging into the MVD EDC containers

### CODE REVIEW

**Review by algattik — APPROVED** [review]

**Review by chrislomonico — APPROVED** [review]
reviewed and approved; great feature to add

File: system-tests/README.md [hunk 1]
```diff
@@ -39,3 +39,9 @@ Run tests:
 ```
 CONSUMER_KEY_VAULT="$CONSUMER_KEY_VAULT" ./gradlew :system-tests:test
 ```
+
+### Debugging MVD locally
+
+Follow the instructions in the previous sections to run an MVD with a consumer and provider locally using docker-compose. 
+
+Once running, you can use the "EDC consumer" or "EDC provider" IntelliJ runtime configurations to remote debug the connector instances.
```
Comment (cpeeyush) [inline system-tests/README.md]: nit: may be we can add link of .run/EDC consumer.run.xml & provider.run.xml here.

**Review by cpeeyush — APPROVED** [review]

File: system-tests/README.md (hunk 1 above)
Comment (marcgs) [reply system-tests/README.md]: These get automatically detected and loaded by IntelliJ, so there is actually no need to know where the runtime configurations are stored IMO

Comment (cpeeyush) [reply system-tests/README.md]: agree but only for version later than 2019.3. But I hope everyone is one the latest.

**Pull request merged**